sites/dotcom/pages/Article.js

```diff
@@ -105,24 +108,11 @@ export default connect('CAPI')(({ CAPI = {} }) => (
                     </Labels>
                 </Cols>
                 <Cols wide={12} leftCol={12}>
-                    <Headline>{CAPI.headline}</Headline>
-                    <Standfirst
-                        dangerouslySetInnerHTML={{
-                            __html: CAPI.standfirst,
-                        }}
-                    />
+                    <Headline />
+                    <Standfirst />
                 </Cols>
             </Row>
-            <div
-                dangerouslySetInnerHTML={{
-                    __html: CAPI.main,
-                }}
-            />
```

Comment:
I guess we don't need `CAPI.main` anymore? I seem to remember it being an empty div in most cases!